migrations/005.do.create-table.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO users (
```

Comment:
What are those new migrations for? Seems to do the same as the first ones

Reply:
yes they are not used I removed them